logos/logos-workernode/tests/test_vllm_process.py

```
@@ -547,14 +560,15 @@ def test_build_cmd_prefers_explicit_lane_context_over_calibrated(monkeypatch) ->
     assert cmd[idx + 1] == "8192"
 
 
-def test_build_cmd_omits_max_model_len_when_no_profile(monkeypatch) -> None:
-    """Without a profile or explicit override, vLLM picks its own default."""
+def test_build_cmd_uses_auto_max_model_len_when_no_profile(monkeypatch) -> None:
+    """Without a profile or explicit override, vLLM sizes the window itself."""
     handle = VllmProcessHandle("lane-test", 19000, OllamaConfig())
     monkeypatch.setattr(handle, "_resolve_vllm_binary", lambda _configured: "/tmp/vllm")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/vllm"

(S108)